rest-api/pom.xml

```diff
@@ -56,6 +56,12 @@
             <version>${workflow.version}</version>
         </dependency>
         <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>pipeline-input-step</artifactId>
+            <version>2.0</version>
+            <optional>true</optional>
```

Comment:
Ok, thanks. So I guess it is safe as this dependency will not be _required_ at installation time as it will not be offered in UC for baselines prior to 1.642.3.